src/Surfnet/ServiceProviderDashboard/Application/Service/EntityMergeService.php

```diff
@@ -177,7 +176,7 @@ private function buildAttributesFromCommand(SaveEntityCommandInterface $command)
             return $attributeList;
         }
 
-        foreach ($this->attributeRepository->findAll() as $definition) {
+        foreach ($this->attributeService->getAttributeTypeAttributes() as $definition) {
```

Comment:
Please create a service method that reads the attributes tailor-made for this application. The entity merge service has nothing to do with the AttributeType form component. So using those value objects here is convenient, but misplaced.

Reply:
Refined that and used an  EntityMergeAttribute valueObject to service the entityMergeService